src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts

```diff
@@ -375,7 +388,15 @@ export class DotEditContentComponent implements OnInit, OnDestroy {
 
     private setPageState(pageState: DotRenderedPageState): void {
         this.pageState = pageState;
-        this.renderPage(pageState);
+        this.showIframe = false;
+
+        setTimeout(() => {
```

Comment:
Please add a comment here: "In order to get the iframe clean up we need to remove it and then re-add it to the DOM"